Sources/CoreMethods/CoreMethods.swift

```diff
@@ -118,4 +131,82 @@ public final class CoreMethods: Sendable {
                 cardID: cardID
             )
     }
+
+    /// Gets the identification document types accepted by the Mercado Pago API
+    ///
+    /// - Returns: An array of ``IdentificationType`` objects representing the available
+    ///   document types for user identification
+    ///
+    /// - Throws:
+    ///   - .invalidURL: If the API endpoint URL is malformed
+    ///   - .decodingFailed(Error): If the response cannot be decoded
+    ///
+    public func identificationType() async throws -> [IdentificationType] {
```

Comment:
should it be identificationTypes?
```suggestion
    public func identificationTypes() async throws -> [IdentificationType] {
```